server/rating_service/rating_service.py

```diff
@@ -158,61 +155,67 @@ async def _get_player_rating(
             result = await conn.execute(sql)
             row = await result.fetchone()
 
-        if not row:
-            return await self._get_player_legacy_rating(player_id, rating_type)
+            if not row:
+                try:
+                    return await self._get_player_legacy_rating(
+                        conn, player_id, rating_type
+                    )
+                except ValueError:
+                    new_rating = await self._create_default_rating(
+                        conn, player_id, rating_type
+                    )
+                    return new_rating
```

Comment:
This could also be a single `return await self._create_default_rating(...` statement, right?

Reply:
Right, I probably just copy/pasted from below

Reply:
Actually I think you wrote this code originally :P

Reply:
:facepalm: